### ISSUE TITLE
Workflow update

### ISSUE THREAD

**Issue by exilesprx** [exilesprx/blockchain#456]
Move audit to end of workflow so builds and tests can occur. If multiple packages have vulnerabilities, as was the case for https://github.com/exilesprx/blockchain/pull/450 and https://github.com/exilesprx/blockchain/pull/455 then dependabot will submit patches for each individually. And therefore the audit job will fail until all are patched. Which leaves no choice but to force merge the individual requests or manually patch the vulnerabilities.

TL;DR: moving the audit job to the end allows audits to continue but ensures builds and tests succeed while allowing individual security patches to be merged with confidence the app won't break
